lib/element.ts

```diff
@@ -414,21 +404,20 @@ export class ElementArrayFinder extends WebdriverWebElement {
    * let list = $$('.items li');
    * expect(list.count()).toBe(3);
    *
-   * @returns {!webdriver.promise.Promise} A promise which resolves to the
+   * @returns {!Promise} A promise which resolves to the
    *     number of elements matching the locator.
    */
-  count(): wdpromise.Promise<number> {
-    return this.getWebElements().then(
-        (arr: WebElement[]) => {
-          return arr.length;
-        },
-        (err: Error) => {
-          if (err instanceof wderror.NoSuchElementError) {
-            return 0;
-          } else {
-            throw err;
-          }
-        });
+  async count(): Promise<number> {
+    try {
+      const arr = await this.getWebElements();
+      return arr.length;
+    } catch (err) {
+      if (err instanceof wderror.NoSuchElementError) {
+        return 0;
+      } else {
+        throw err;
```

Comment:
why not `throw new Error(err)`? I read that this is better practice :)

Reply:
Without a try catch, this will automatically throw the `err`. Wrapping the `err` in a new `Error` could give us the wrong stack trace and error message. When returning a number of elements, we are just catching where there are no elements to return a count of 0.

I think it is okay to leave this as is.